src/viem/prepareTransactions.ts

```diff
@@ -9,18 +10,18 @@ import { publicClient } from 'src/viem/index'
 import {
   Address,
   EstimateGasExecutionError,
+  ExecutionRevertedError,
   InsufficientFundsError,
   encodeFunctionData,
-  ExecutionRevertedError,
 } from 'viem'
-import stableToken from 'src/abis/StableToken'
 
 const TAG = 'viem/prepareTransactions'
 
 export interface PreparedTransactionsPossible {
   type: 'possible'
   transactions: TransactionRequestCIP42[]
   feeCurrency: TokenBalance
+  maxGasFeeInDecimal: BigNumber
```

Comment:
Would be nice to use the same property name here and in
https://github.com/valora-inc/wallet/blob/41eae0760b73f3ac7b5072907bf852813e7ba3e8/src/viem/prepareTransactions.ts#L28

Reply:
yes good idea...this wasn't there when i started implementing this PR. naming is hard, and i'm not sure that `InDecimal` is crystal clear but it seems like we have 1 gas fee value in the minimum token units and another after shifting by the token decimals. `maxGasFee` there is actually in decimals too :/ so I thought it was more clear to rename `maxGasFee` to `maxGasFeeInDecimal`. wdyt?